### PR TITLE
Remove justfile & fully replace remaining commands with pixi

### LINKED ISSUE #5595: Remove `justfile`
And replace it with `pixi.toml`. We also need to take a pass on the names of the commands in `pixi.toml` to make them consistent.

### PR DESCRIPTION
**Pull request by Wumpf**
### What

* Fixes #5595

Has a few judgment calls on what is no longer needed, but preserves most of what we had.
Notably removes `py-requirements` command for the time being - the hope is that #5890 will fix things. (porting it proved difficult)
See
* https://github.com/rerun-io/rerun/issues/5894

Other things in here:
* taplo (toml formatting) was very slow since despite extensive 'exclude' list it looked at too many files. Turned it around to 'include' things and it's a LOT faster now
* Updated Ruff & removed last use of blackdoc. The only thing we're not properly catching with this now is too long lines, I figured it's not worth keeping it around just for that.
* various cleanups of pixi.toml file: trying to make commands a bit more consistent overall

Tested on:
* [x] mac
* [x] windows

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using newly built examples: [rerun.io/viewer](https://rerun.io/viewer/pr/5892)
  * Using examples from latest `main` build: [rerun.io/viewer](https://rerun.io/viewer/pr/5892?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [rerun.io/viewer](https://rerun.io/viewer/pr/5892?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!

- [PR Build Summary](https://build.rerun.io/pr/5892)
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

### CODE REVIEW

File: pixi.toml [hunk 1]
```diff
@@ -72,41 +78,61 @@ rerun-web-release = { cmd = "cargo run --package rerun-cli --no-default-features
...
+toml-fmt = "taplo fmt"
+
+# Misc formatting tasks.
 misc-fmt = "prettier --write '**/*.{yml,yaml,js,css,html}'"
 misc-fmt-check = "prettier --check '**/*.{yml,yaml,js,css,html}'"
+
 ruff-fmt = "ruff format --config rerun_py/pyproject.toml ."
-ruff-fix = "ruff --fix --config rerun_py/pyproject.toml ."
-toml-fmt = "taplo fmt"
+ruff-fix = "ruff check --fix --config rerun_py/pyproject.toml ."
```
Comment (emilk) [inline pixi.toml:106]: Can we make commands "private" somehow? The user should focus on `py-fmt` I think, not this implementation detail. Maybe just move it to later in the file?

(I think commands should be high level, and not focus on how they are implemented, i.e. "ruff")

File: rerun_py/pyproject.toml [hunk 2]
```diff
@@ -47,6 +47,8 @@ rerun = "rerun_bindings:main"
 
 # target-version = "py38"  # inferred from requires-python, see https://beta.ruff.rs/docs/settings/#target-version
 
+unsafe-fixes = true
```
Comment (emilk) [inline rerun_py/pyproject.toml:50]: can you add a comment explaining what this is, so the reader (me) don't have to google it? 🙏

**Review by emilk — APPROVED** [review]
Fan-fucking-tastic!

File: pixi.toml (hunk 1 above)
Comment (Wumpf) [reply pixi.toml:106]: > I think commands should be high level

agreed!
Not aware of a way to hide commands - we could also just make the py command combine these.

**Pull request merged**